compose/project.py

```diff
@@ -695,7 +695,9 @@ def push(self, service_names=None, ignore_push_failures=False):
             repo, tag, sep = parse_repository_tag(service.image_name)
             service_image_name = sep.join((repo, tag)) if tag else sep.join((repo, 'latest'))
 
-            if service_image_name not in unique_images:
+            # Add service to unique if it's pushable
```

Comment:
This comment is redundant; it's an exact "copy" of the code below (only in a sentence)